### ISSUE TITLE
YouTube links 

### ISSUE THREAD

**Issue by Yoshi2023** [tuyafeng/Via#550]
Links from the YouTube app cannot be opened

**Comment by lfom** [comment]
Yeah, for some reasom Via isn't offered as an option to open youtu.be/youtube.com redirection links from YouTube app.

**Comment by gbdmpl** [comment]
that will be nice function
